crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -26,12 +26,27 @@ pub enum AppRequest {
 
     /// Call a zome function.
     ///
-    /// See [`ZomeCall`] to understand the data that must be provided.
+    /// The payload to this call is composed of the serialized [`ZomeCallParams`] as bytes
+    /// and the provenance's signature.
+    ///
+    /// Serialization must be performed with MessagePack. The resulting bytes are hashed using the
+    /// Blake2b 256-bit algorithm, and the hash is signed with the provenance's private ed25519 key.
+    /// The hash is not included in the call's payload.
```

Comment:
Yay, docs! Pretty sure that's the first time the zome call signing has been documented at all :)